tests/regression/36-apron/38-branch-global.c

```diff
@@ -1,10 +1,15 @@
-// SKIP PARAM: --set ana.activated[+] apron --set ana.path_sens[+] threadflag --enable ana.int.interval
```

Comment:
I am a bit worried that this changes what the test tests. Now there is a concrete upper bound upper available, the  bug in the apron analysis may not be triggered anymore...

If this test is also helpful here, it may make sense to duplicate it.

Reply:
There was also a concrete upper bound before. Being global, `ARR_SIZE` was implicity 0, so the test was completely trivial by even def_exc, not to mention interval (which is also enabled for no reason) and apron itself.

This test is for #317 and it still checks that apron doesn't crash on global variables in branch expressions. But now it additionally checks that it actually refines them as well.

Reply:
Ah, yes, I didn't realize it was a global.